### DISCUSSION TITLE
【功能建议】节能模式希望能够设置不降低屏幕亮度

### DISCUSSION THREAD

**Discussion by widon1104** [linuxdeepin/developer-center#2982] [Features Request & Ideas | 特性请求 & 头脑风暴]
![截图录屏_选择区域_20210122205934](https://user-images.githubusercontent.com/12093211/105493947-0248cb00-5cf5-11eb-932d-d70bbaa086a6.jpg)
我使用的deepinV20.1，想切换为节能模式仅仅是希望能降低cpu频率从而降低cpu功耗，从而降低cpu温度（我使用的无风扇mini主机，靠外壳散热，cpu温度太高了外壳会非常热）。这个时候把屏幕亮度给我降低我就尴尬了，我用的可是台式机，不缺那点电。

希望可以设置为节能模式，降低cpu温度，同时设置不降低屏幕亮度。

**Comment by meiyixiang** [comment]
你可以升级系统，现在对于电源管理提供了许多其它的功能管理项目
